src/features/content-explorer/content-explorer-modal/ContentExplorerModal.js

```diff
@@ -14,6 +14,7 @@ type Props = {
     description?: string,
     isOpen?: boolean,
     onRequestClose?: Function,
+    onSelectedClick?: Function,
```

Comment:
nit: I know it's not being done above, but we could use more explicit type for this new handler

Reply:
updated